snap/snapcraft.yaml

```diff
@@ -1,12 +1,13 @@
 name: reicast
 version: git
-summary: Reicast
+summary: Reicast -- The fastest Dreamcast Emulator
```

Comment:
Do we really want to state that?

Reply:
Yup ;) We are and we take pride in it

Reply:
It's also only stated in the snapcraft yaml, so nonarcissistic harm done :)